core/planner.c

```diff
@@ -565,15 +565,16 @@ bool trial_ascent(int wait_time, int trial_depth, int stoplevel, int avg_depth,
 	// For consistency with other VPM-B implementations, we should not start the ascent while the ceiling is
 	// deeper than the next stop (thus the offgasing during the ascent is ignored).
 	// However, we still need to make sure we don't break the ceiling due to on-gassing during ascent.
-	cache_deco_state(&trial_cache);
+	cache_deco_state(ds, &trial_cache);
 	if (wait_time)
-		add_segment(depth_to_bar(trial_depth, dive),
+		add_segment(ds, depth_to_bar(trial_depth, dive),
 			    gasmix,
 			    wait_time, po2, dive, prefs.decosac);
-	if (decoMode() == VPMB && (deco_allowed_depth(tissue_tolerance_calc(dive,
-										 depth_to_bar(stoplevel, dive)),
-							   surface_pressure, dive, 1) > stoplevel)) {
-		restore_deco_state(trial_cache, false);
+	if (decoMode() == VPMB && (deco_allowed_depth(tissue_tolerance_calc(ds, dive,
+									    depth_to_bar(stoplevel, dive)),
+						      surface_pressure, dive, 1)
+				   > stoplevel)) {
```

Comment:
Coding style

Reply:
I tried to improve these. Some where caused by ds originally being called deco_state. But in order to somewhat stay within reasonable number of columns there have to be breaks and in the following line we need enough indentation.